### ISSUE TITLE
Support

### ISSUE THREAD

**Issue by bethuxs** [victorximenis/omnipay-mercadopago#1]
Hi, is this project actived?

**Comment by victorximenis** [comment]
Yes. The project has been idle but I'm coming back now

**Comment by bethuxs** [comment]
Thank you for your response. is this repository completely functional?

**Comment by victorximenis** [comment]
Yeah, the features that i've implemented. But has few other things that i want to add.

**Comment by bethuxs** [comment]
Ok. I'm trying to implement in a project. But I don't understand something. Actually I'm redirecting to MercadoPago page and coming back to my site with the URL return.

**Comment by victorximenis** [comment]
What exactly you dont understand?

**Comment by bethuxs** [comment]
Is this a transparent gateway?

**Comment by pckz** [comment]
Hello! Same here, I just started to add mercadopago gateway to omnipay lib to my project, specifically  to laravel 5.1.

What payments method/features are active and which ones do you want to implement?
And second.. do you have a example code to use as guide? 

Thanks you very much victor! regards from chile :D
